dentapp/src/App.js

```diff
@@ -1,26 +1,31 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
+import React from 'react';
+import { BrowserRouter, Switch, Route } from "react-router-dom";
+
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap/js/dist/index.js';
+
+import Header from './components/Header'
+import LoginLogo from './components/LoginLogo';
+import LoadingSpinner from './components/LoadingSpinner';
+
+
+function App(props) {
+
+  return (
+    <BrowserRouter>
+      <Header backgroundColor="#00D9CC" color="#ffffff" handleButtonPress="/">
+        <LoginLogo />
+      </Header>
+
+      <Switch>
+        <Route path="/">
+          <div>Registro</div>
+          <LoadingSpinner handleShow={true}/>

```

Comment:
instead of handleShow, call it just show, handleShows will be more proper for a function, also you can put the prop like this:           
<LoadingSpinner show/>
<LoadingSpinner show={true}/>

Both are the same